packages/phalanx/CMakeLists.txt

```diff
@@ -14,12 +14,13 @@ TRIBITS_ADD_SHOW_DEPRECATED_WARNINGS_OPTION()
 
 TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ALLOW_MULTIPLE_EVALUATORS_FOR_SAME_FIELD
   PHX_ALLOW_MULTIPLE_EVALUATORS_FOR_SAME_FIELD
-  "Enable to use backwards compatible mode where multiple evaluators can be registered that evalaute the same field. Phalanx will use the first evaluator it finds. This should default to off but for backwards compatibility with current codes, we default this to ON."
-  ON )
+  "Enable to use backwards compatible mode where multiple evaluators can be registered that evalaute the same field. Phalanx will use the first evaluator it finds. Default to OFF but unless Panzer is enabled. For backwards compatibility with Panzer users, we default this to ON."
+  ${PROJECT_NAME}_ENABLE_Panzer )
```

Comment:
@rppawlo, 

the above always evaluates to `TRUE` which is not what you want.

This needs to be changed to:

```
"${${PROJECT_NAME}_ENABLE_Panzer}"
```

with the quotes for the reason described in:

* https://tribits.org/doc/TribitsDevelopersGuide.html#cmake-language-overview-and-gotchas

(see `EMPTY_VAR`)

The issue is that TriBITS "ENABLE" vars can be `ON` or `OFF` or empty "".    For the latter case, you need the quotes.

Reply:
@rppawlo, let me pull the branch and check this real quick.